src/Parser/FrostMage/Modules/Items/ZannesuJourney.js

```diff
@@ -0,0 +1,45 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import getDamageBonus from '../MageCore/GetDamageBonus';
+
+const DAMAGE_BONUS = .35;
+
+class ZannesuJourney extends Module {
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+  damage = 0;
+  stackCount = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasWaist(ITEMS.ZANNESU_JOURNEY.id);
+  }
+
+  on_byPlayer_cast(event) {
+    if (event.ability.guid !== SPELLS.BLIZZARD.id) {
+      return;
+    }
+    const buff = this.combatants.selected.getBuff(SPELLS.ZANNESU_JOURNEY_BUFF.id,event.timestamp);
```

Comment:
While this will work correctly, `getBuff` defaults to using the current timestamp, so you don't actually need to include the 2nd parameter.

Reply:
im getting warnings because of the line beneath it that has && and || in the same line, but thats the only way i could get it to count correctly without it telling me that stacks is undefined.

Reply:
nevermind, figured it out

Reply:
I'm not talking about the line beneath it, I'm talking about this line right here. You can leave out the second parameter, `event.timestamp`

Reply:
Yea, i did remove that, i should have made a separate comment instead of adding that to this one